video_reader.py

```diff
@@ -0,0 +1,21 @@
+__author__ = "Keenan Manpearl"
+__date__ = "2023/03/01"
+
+"""
+Reads in an avi video file and returns an array
+"""
+
+import cv2
+import numpy as np
+
+
+def read_video(video_path, n_frames=10):
```

Comment:
I think it would be fine to include this short of a function in the file you are using it in, but probably don't change it for this PR.